frontend/src/libs/components/card/styles.module.scss

```diff
@@ -81,6 +108,10 @@
   .no-image {
     padding-left: 20px;
   }
+
+  .icon-right {
+    left: 190px;
```

Comment:
Will this work for buttons with different width?

Reply:
@Liza-Veis yep